### DISCUSSION TITLE
Best way to refresh Directus #main_content?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15854] [Archive]
Is there a way to refresh smartly the #main_content without refreshing the whole page?
It would be nice to be able to use the `Key-Changing Technique` as described here [The correct way to force Vue to re-render a component](https://michaelnthiessen.com/force-re-render/)

_I'd like to refresh the #main_content after some actions of the user on Interface / Display extensions that modifies the DB thorough an API._

**Comment by br41nslug** [comment]
Currently there's only the "automatic refresh" feature which will refresh the data without reloading. 
![image](https://user-images.githubusercontent.com/9389634/193560372-d693c77b-ea4b-40a0-b3c1-0f92867b335f.png)
Once websockets are implemented smarter updates to the content can be supported.

**Reply by Pachat** [reply]
Good news :)

Meanwhile, I'll use the "horrible" way described in [The correct way to force Vue to re-render a component](https://michaelnthiessen.com/force-re-render/)
